src/main/java/com/questhelper/quests/thedepthsofdespair/TheDepthsOfDespair.java

```diff
@@ -0,0 +1,280 @@
+		superEnergyOrStamina = new ItemRequirement("Super Energy or Stamina potions", -1, -1);
+
+		dramenStaff = new ItemRequirement("Access to Fairy Rings", ItemID.DRAMEN_STAFF);
+		dramenStaff.addAlternates(ItemID.LUNAR_STAFF);
+
+		foodIfLowLevel = new ItemRequirement("Food (if low level)", -1, -1);
+		foodIfLowLevel.setDisplayItemId(BankSlotIcons.getFood());
+
+		weapon = new ItemRequirement("A Weapon", -1, -1);
+		weapon.setDisplayItemId(BankSlotIcons.getCombatGear());
+
+		varlamoreEnvoy = new ItemRequirement("Varlamore Envoy", ItemID.VARLAMORE_ENVOY);
+		varlamoreEnvoy.setHighlightInInventory(true);
+
+		royalAccordOfTwill = new ItemRequirement("Royal Accord of Twill", ItemID.ROYAL_ACCORD_OF_TWILL);
+	}
+
+	public void loadZones()
+	{
+		houseWestOfVinery = new Zone(new WorldPoint(1810, 3550, 0), new WorldPoint(1820, 3560, 0));
+		arceuusLibraryF1 = new Zone(new WorldPoint(1607, 3831, 0), new WorldPoint(1658, 3784, 0));
+		arceuusLibraryF2 = new Zone(new WorldPoint(1607, 3831, 1), new WorldPoint(1658, 3784, 1));
+		arceuusLibraryF3 = new Zone(new WorldPoint(1607, 3831, 2), new WorldPoint(1658, 3784, 2));
+		crabclawCaves = new Zone(new WorldPoint(1644, 9851, 0), new WorldPoint(1728, 9793, 0));
+		crabclawCavesPart1A = new Zone(new WorldPoint(1644, 9851, 0), new WorldPoint(1722, 9823, 0));
+		crabclawCavesPart1B = new Zone(new WorldPoint(1665, 9823, 0), new WorldPoint(1707, 9813, 0));
+		crabclawCavesPart2 = new Zone(new WorldPoint(1703, 9821, 0), new WorldPoint(1724, 9792, 0));
+		crabclawCavesPart3 = new Zone(new WorldPoint(1688, 9804, 0), new WorldPoint(1703, 9794, 0));
+		crabclawCavesDownstairs = new Zone(new WorldPoint(1670, 9763, 0), new WorldPoint(1701, 9742, 0));
+	}
+
+	public void setupConditions()
+	{
+		inVineryHouse = new ZoneCondition(houseWestOfVinery);
+		inArceuusLibrary = new ZoneCondition(arceuusLibraryF1, arceuusLibraryF2, arceuusLibraryF3);
+		inCaves = new ZoneCondition(crabclawCaves);
+		inFirstAreaOfCaves = new ZoneCondition(crabclawCavesPart1A, crabclawCavesPart1B);
+		inSecondAreaOfCaves = new ZoneCondition(crabclawCavesPart2);
+		inThirdAreaOfCaves = new ZoneCondition(crabclawCavesPart3);
+
+		ZoneCondition cavesDownstairs = new ZoneCondition(crabclawCavesDownstairs);
+		downstairsInCaves = new Conditions(cavesDownstairs, new InInstanceCondition());
+
+		hasTheVarlamoreEnvoy = new ItemRequirementCondition(varlamoreEnvoy);
+	}
+
+	public void setupSteps()
+	{
+		talkToLordKandur = new NpcStep(this, NpcID.LORD_KANDUR_HOSIDIUS, new WorldPoint(1782, 3572, 0),
+			"Talk to Lord Kandur Hosidius in the house west of the vinery.");
+		talkToLordKandur.addDialogSteps("Anything I can help you with?", "Yes.");
+
+		talkToChefOlivia = new NpcStep(this, NpcID.CHEF_OLIVIA, new WorldPoint(1776, 3567, 0),
+			"Speak to Chef Olivia in Lord Kandur's kitchen.");
+
+		talkToGalana = new NpcStep(this, NpcID.GALANA, new WorldPoint(1649, 3824, 0),
+			"Speak to Galana.");
```

Comment:
I'd personally specify 'Speak to Galana in the Arceuus library' so there's context as to where you're going next.